src/GraphbackServer.ts

```diff
@@ -61,6 +60,16 @@ export class GraphbackServer {
     return `http://localhost:${this.serverPort}${ENDPOINT}`;
   }
 
+  public getSubscriptionsUrl(): string {
```

Comment:
We can call it just `wsUrl`, wdyt?
```suggestion
  public getWsUrl(): string {
```

Reply:
@b1zzu I don't like `wsUrl` because it can be used to describe any web socket url, do you have another option?

Reply:
I feel `wsUrl` match `httpUrl` because otherwise if we want to unify the names we should rename `httpUrl` to something like `queriesMutationsUrl` :confounded:

Reply:
And I can't think of any other options :sweat_smile:

Reply:
I get it and make sense XD, I'll change the name